src/index.js

```diff
@@ -331,6 +337,7 @@ function createConfig(options, entry, format, writeMeta) {
 						},
 					}),
 					!useTypescript &&
```

Comment:
seems like we could create a single variable with enumerable values, smth like `language`? that we could reuse in all those conditional places here, seems slightly better than creating a boolean variable for each "language"

Reply:
I want to attempt this after #198. My feel is the extension checks lack any encapsulation leading to #193 and #196. May I refactor in for #141 or separately? Have you already thought about that @Andarist @developit?

Reply:
@texastoland Feel free to continue on my PR if it makes sense. (not finding a lot of time lately due to multiple factors)